.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Describe the problem
+
+## Describe the solution
+
+## Risk & Testing considerations
+What is the risk level of this change (Low / Medium / High)? For Medium / High please elaborate.
```

Comment:
A link to the risk level definition is to be added (once ready).

Reply:
Are we planning on hosting that risk definition on the internal Wiki? Or google docs we won't be able to link to non-public facing documentation here.

Reply:
I was thinking of using a public wiki to enable external contributors to see the contents.

Reply:
I agree a public location is the way to go. How about in GitHub, either as a wiki page, or even just as another markdown file in a known location in our source tree (and therefore having a consistent link)?

Reply:
Both (Github wiki and another md file) sound good to me. Thanks for the suggestions, @jeremythorp.

Reply:
Sounds like we're agreed on a MD file or github wiki page. Luke feel free to do what works best.